### DISCUSSION TITLE
Use HDMI for audio out

### DISCUSSION THREAD

**Discussion by astralaster** [paulscottrobson/neo6502-firmware#191] [Ideas]
Maybe its possible to use HDMI audio. There are some infos here: https://github.com/Wren6991/PicoDVI/issues/5

**Comment by paulscottrobson** [comment]
I'm monitoring this. If it will work it's an great solution - no extra hardware required. It does have some core impact though according to the fork README. How much is difficult to know. You'd hope that (say) four simple waveforms should be okay.

**Comment by paulscottrobson** [comment]
Think we're now looking at PWM options.

**Comment by astralaster** [comment]
[sprite_bounce_audio_uf2.zip](https://github.com/paulscottrobson/neo6502-firmware/files/14911403/sprite_bounce_audio_uf2.zip)
Try this firmware. It displays sprites in our video mode while outputting a beep in stereo over HDMI.
